mintpy/closure_phase_bias.py

```diff
@@ -131,35 +672,156 @@ def main(iargs = None):
                 print('box width:  {}'.format(box_width))
                 print('box length: {}'.format(box_length))
 
-            closurephase[box[1]:box[3],box[0]:box[2]], numcp = cum_seq_closurePhase(SLC_list, date12_list_all, inps.ifgram_stack, ref_phase,inps.nl,box)
-
-
-
+            closurephase[box[1]:box[3],box[0]:box[2]], numcp = sum_seq_closure_phase(slc_list, date12_list_all, ifgram_stack, ref_phase,nl,box)
     # What is a good thredshold?
     # Assume that it's pure noise so that the phase is uniform distributed from -pi to pi.
     # The standard deviation of phase in each loop is pi/sqrt(3) (technically should be smaller because when forming loops there should be a reduction in phase variance)
-    # The standard deviation of phase in cumulative wrapped closure phase is pi/sqrt(3)/sqrt(num_cp) -- again another simplification assuming no correlation.
-    # We use 3\delta as default threshold -- 99.7% confidence
+    # The standard deviation of phase in cumulative wrapped closure phase is pi/sqrt(3)/sqrt(numcp) -- again another simplification assuming no correlation.
+    # We use 3\delta as threshold -- 99.7% confidence
 
-    if inps.numsigma:
-        threshold_cp = np.pi/np.sqrt(3)/np.sqrt(numcp)*inps.numsigma
-    else:
-        threshold_cp = np.pi/np.sqrt(3)/np.sqrt(numcp)*3 # 3/sigma, 99.7% confidence
+    threshold_pha = np.pi/np.sqrt(3)/np.sqrt(numcp)*num_sigma
 
     mask = np.ones([length,width],np.float32)
-    mask[np.abs(np.angle(closurephase))>threshold_cp] = 0 # this masks areas with potential bias
-    mask[np.abs(np.abs(closurephase)/numcp < inps.episilon)] = 1 # this unmasks areas with low correlation (where it's hard to know wheter there is bias either)
+    mask[np.abs(np.angle(closurephase))>threshold_pha] = 0 # this masks areas with potential bias
+    mask[np.abs(np.abs(closurephase)/numcp < threshold_amp)] = 1 # this unmasks areas with low correlation (where it's hard to know wheter there is bias either)
 
     # save mask
     meta = dict(stack_obj.metadata)
     meta['FILE_TYPE'] = 'mask'
     ds_name_dict = {'cpmask': [np.float32, (length, width), mask],}
-    writefile.layout_hdf5(os.path.join(inps.outdir,'cpmask.h5'), ds_name_dict, meta)
+    writefile.layout_hdf5(os.path.join(outdir,'cpmask.h5'), ds_name_dict, meta)
```

Comment:
How about naming this mask file as `maskClosurePhase.h5` to be more readable? It will also be more consistent with other names, e.g. `maskTempCoh.h5`.

Reply:
Thanks! Modified as suggested.